### ISSUE TITLE
Allow the usage of System Design Language (sdl) shapes 

### ISSUE THREAD

**Issue by SoerenGreule** [magjac/graphviz-visual-editor#149]
There is a great set of sdl-shapes available that can be used in graphviz as [External PostScript procedure](https://graphviz.org/faq/#ext_ps_proc). Would it be possible to make these available in the editor?

The shapes (shown below) were developed by Mark Rison of CSR and can be found in the contrib/sdlshapes directory within the graphviz package.
![](https://graphviz.org/doc/info/sdlshapes.png)

**Comment by magjac** [comment]
I'm not sure, but I don't think so. The editor is based on [d3-graphviz](https://github.com/magjac/d3-graphviz) which operates on SVG output only and from what I understand you have to use PostScript output for the SDL shapes.

**Comment by SoerenGreule** [comment]
Yes, using PostScript seems to be a requirement. Would you be able/willing to add them when I redraw them as svgs?

**Comment by magjac** [comment]
I don't think Graphviz supports user defined shapes in any other format than PostScript so I don't think it's possible to do it that way. 

Perhaps it's possible to use [external images](https://graphviz.org/faq/#ext_image) in some way through the [image](https://graphviz.org/doc/info/attrs.html#d:image) attribute. From what I can see it supports `.ps` files too. I'm uncertain however how the user interface should be in that case since the image attribute is part of the DOT source that the user supplies while I guess that the images themselves should be hosted by the editor application.
